mailer.rst

```diff
@@ -129,6 +129,13 @@ Service               Install with
     <mail-catcher>` instead. Note that most supported providers also offer a
     free tier.
 
+.. note::
+
+    When using a 3rd party transport, you might want to use a web API
+    instead of directly rely on the SMTP protocol to send emails.
+    To do so, you have to install the optional HttpClient dependency
+    thanks to ``composer require symfony/http-client``.
```

Comment:
We should point out that the HttpClient does not replace the bridge, but in fact needs to be installed in addition to it.

Reply:
Rephrased, is this better?